### PR TITLE
bump tempo image to 2.0.0

### PR DESCRIPTION
**Pull request by rubenvp8510**
Search cannot be enabled or disabled anymore, I removed this from the CR but this will break the previous version of the CR, because we haven't released yet. Is this ok?

### CODE REVIEW

**Review by andreasgerstmayr — CHANGES_REQUESTED** [review]
LGTM, two minor things should be reverted

File: bundle/manifests/tempo-operator.clusterserviceversion.yaml [hunk 1]
```diff
@@ -484,7 +481,7 @@ spec:
                 - --config=controller_manager_config.yaml
                 command:
                 - /manager
-                image: ghcr.io/os-observability/tempo-operator/tempo-operator:v0.0.1
+                image: ghcr.io/rvargasp/tempo-operator/tempo-operator:v0.0.1
```
[the remark this answers is not in the document]
Comment (rubenvp8510) [reply bundle/manifests/tempo-operator.clusterserviceversion.yaml:484]: oh yeah sorry.

File: config/manager/kustomization.yaml [hunk 2]
```diff
@@ -12,5 +12,5 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 images:
 - name: controller
-  newName: controller
+  newName: quay.io/rvargasp/tempo-operator
```
[the remark this answers is not in the document]
Comment (rubenvp8510) [reply config/manager/kustomization.yaml:15]: :D

**Review by iblancasa — CHANGES_REQUESTED** [review]
Please, modify the default value for the `TEMPO_VERSION` Makefile variable

File: bundle/manifests/tempo-operator-manager-config_v1_configmap.yaml [hunk 3]
```diff
@@ -23,7 +23,7 @@ data:
     # after the manager stops then its usage might be unsafe.
     # leaderElectionReleaseOnCancel: true
     images:
-      tempo: docker.io/grafana/tempo:1.5.0
+      tempo: docker.io/grafana/tempo:2.0.0
```
Comment (frzifus) [inline bundle/manifests/tempo-operator-manager-config_v1_configmap.yaml:26]: Do we need to update tempo-query then too?

**Comment by andreasgerstmayr** [comment]
> Please, modify the default value for the `TEMPO_VERSION` Makefile variable

Is this variable used anywhere?

**Comment by frzifus** [comment]
> Is this variable used anywhere?

nope

**Comment by iblancasa** [comment]
> > Please, modify the default value for the `TEMPO_VERSION` Makefile variable
> 
> Is this variable used anywhere?

I forget to remove it in the PR for the release. I can modify on the PR with the release process

File: bundle/manifests/tempo-operator-manager-config_v1_configmap.yaml (hunk 3 above)
Comment (rubenvp8510) [reply bundle/manifests/tempo-operator-manager-config_v1_configmap.yaml:26]: It will be good I think

Comment (rubenvp8510) [reply bundle/manifests/tempo-operator-manager-config_v1_configmap.yaml:26]: I won't do it, since the githash you need is not included in 2.0.0, is 2.0.0+

**Review by andreasgerstmayr — APPROVED** [review]

**Review by frzifus — APPROVED** [review]

**Pull request merged**